### DISCUSSION TITLE
Co-embedding RNA and ATAC in UMAP space

### DISCUSSION THREAD

**Discussion by jindalk** [GreenleafLab/ArchR#672] [Feature Requests]
Hi, is there anyway to plot a co-embedded UMAP after integrating RNA and ATAC datasets (not from a mulitomic assay) using addGeneIntegrationMatrix?

I tried forcing the multiome vignette on this problem but it needs same cell barcodes across the 2 assays which I do not have.

Thanks

**Comment by rcorces** [comment]
I dont _think_ this would make biological sense but maybe I'm mis-interpreting. But given that you are inferring RNA expression from gene scores and using CCA to transfer the RNA data into your ATAC cells, I think that doing a co-embedding on this would be problematic. Perhaps you can clarify why you want to do this and what gain you expect over an embedding based solely off of ATAC-seq information?

**Reply by jindalk** [reply]
Hi, I was mostly just trying to create a plot like the one shown in the co-embedding section of this vignette from Signac: https://satijalab.org/seurat/archive/v3.0/atacseq_integration_vignette.html

But I agree that this might have little biological significance and is mostly just a cool visualisation trick haha!

Thanks

**Reply by rcorces** [reply]
Oh I see what you're trying to do. Yes - for visualization purposes this makes sense. Currently ArchR doesnt have this feature but it could be worthwhile to implement. I'm going to mark this as an "Idea" (aka feature request)
